src/utils_begin.jl

```diff
@@ -324,19 +324,20 @@ Determines what the maximum memory is based on the device type (apple, windows,
 function localhost_memory()
     if Sys.isapple()
         cmd = `sysctl hw.memsize` # for OSX
-        mem_size = parse(Int, match(r"\d+", readchomp(cmd)).match) / 1024^3
+        mem_size = parse(Int, match(r"\d+", readchomp(cmd)).match) / 1024^3 # GiB
     elseif Sys.isunix()
```

Comment:
I am a bit worried about this `Sys.isunix()` test before `Sys.islinux()` test

Reply:
Nothing was changed with respect to that (just a comment that this in GiB)

Reply:
ok, but should we move the `Sys.isunix()` check as the last option?

I fear that a Linux system might fall into this if statement and not go in the `Sys.islinux()`